src/types/plugin-inputs.ts

```diff
@@ -45,9 +45,9 @@ export const pluginSettingsSchema = T.Object({
   approvalsRequired: approvalsRequiredSchema,
   mergeTimeout: mergeTimeoutSchema,
   /**
-   * The location of the database
+   * The list of organizations or repositories to watch for updates.
    */
-  databaseUrl: T.String({ default: "database/sql.db" }),
+  watch: T.Array(T.String({ minLength: 1 }), { default: [] }),
```

Comment:
ideally this plugin should be at the org level config, right? but I'm not sure how are we merging repo and org config

Reply:
There is a lot of ongoing discussions on how to merge configuration at the moment, so I am not sure myself. Since it supports excluding repositories too, it should be enough for now?